### ISSUE TITLE
Does it alert to offline deaths of those on watchlist?

### ISSUE THREAD

**Issue by ghost** [noctellios/RipBozo#1]
As I understand it, the watchlist only reacts via callback registered with DeathNotificationLib_HookOnNewEntry, which only calls back on new deaths while live.

**Comment by noctellios** [comment]
Correct, it relies on the NewEntry broadcast. There are some references in Deathlog proper about attempting to get some recorded data from guildmembers' recordings when you log in, but I'm not sure if it was fully implemented over there. Even if it was, from there it would need to trigger the hook for RipBozo to trigger and I doubt they'd risk a chat spam disconnect attempting to send historical data.
